inspec.yml

```diff
@@ -780,11 +786,11 @@ inputs:
       "release key 2": "567E 347A D004 4ADE 55BA  8A5F 199E 2F91 FD43 1D51"
       "auxiliary key": "7E46 2425 8C40 6535 D56D  6F13 5054 E4A4 5A63 40B3"
 
-  # SV-258068, SV-258077
+  # SV-258077
   - name: stop_idle_session_sec
     description: Number of seconds the system can be idle before session timeout
     type: Numeric
-    value: 900
+    value: 600
```

Comment:
Checked the STIG (v2R3) and the default STIG value should still be 900 for this one.